src/Paprika/Data/NibblePath.cs

```diff
@@ -545,7 +545,11 @@ public bool Equals(in NibblePath other)
 
         return FindFirstDifferentNibble(other) == Length;
     }
-
+    public bool StartsWith(in NibblePath other)
+    {
+        return FindFirstDifferentNibble(other) == Length;
```

Comment:
I think it's wrong. It should assert whether the first different is `other.Length` as `ABC` starts with `AB` only if the first different is `2` as it's `AB` length.

Reply:
Done